### ISSUE TITLE
Accessibilty: logical grouping of activity should be role="group" instead of "region"

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3092]
https://github.com/microsoft/BotFramework-WebChat/commit/2b77811796e3fcee769140ba54829df7db8f7f11

![image](https://user-images.githubusercontent.com/1622400/78699526-e59feb80-78b8-11ea-8d4b-7763ea104723.png)
